data/campaigns/Heir_To_The_Throne/scenarios/18_A_Choice_Must_Be_Made.cfg

```diff
@@ -289,7 +289,7 @@
             [then]
                 [message]
                     speaker=unit
-                    message= _ "Prince Konrad, you and your friends keep talking about avoiding the Arkan-thoria. But we can lead you down this river. With the help of us merfolk, you could reach the home of the North Elves much sooner by the river road, rather than by taking either one of the land routes. I can not promise you that it will be a safe journey, because I do not know what lies ahead, but it is an option and you should consider it, my lord."
+                    message= _ "Prince Konrad, you and your friends keep talking about avoiding the Arkan-thoria. But we can lead you down this river. With the help of us merfolk, you could reach the home of the North Elves much sooner by the river road than by taking either one of the land routes. I can not promise you that it should be a safe journey, because I know not what lies ahead, but it is an option and you should consider it, my lord."
```

Comment:
Since you're editing this anyway, maybe fix the `can not`? It should be one word.

Reply:
Which `should`? There are two. The first one is indirect speech for "I can't promise 'it shall be a safe journey'". The second means "ought to".

I'll fix the "can not".

Reply:
Both cannot and can not are correct. OED says cannot is just more usual. In this case I prefer the more unusual form because it's coming from a merman and accentuates that they're not native speakers of the language.

I would say "can not promise it will be a safe journey" or "can not promise, but it should be a safe journey", and prefer the latter myself.

Reply:
I was referring to the "should" that you just added.

Reply:
@GregoryLundberg Saying "but it should be a safe journey" would change the meaning of the sentence, wouldn't it? Did you mean without the word "but"?

Thanks for looking up "can not", I also prefer the more unusual form.

Reply:
I prefer the usual form in this case.

Reply:
Hmm. Looking at it, put "will" back. It's future tense. But as I read it, part of the problem is comma splicing. Maybe it would be more apparent what to do if you broke it into two or three sentences and maybe rearranged the concepts for a more clear chain of though.

Reply:
I've pushed two more fixes in a separate commit. Trying to rearrange:

> Prince Konrad, you and your friends keep talking about avoiding the Arkan-thoria. Would we not better follow the river? We merfolk can lead you downstream; with our help, you could reach the home of the North Elves much sooner by the river road than by either one of the land routes. While I know not what lies ahead and can not promise it will be a safe journey, it is an option for you to consider, my lord.

@CelticMinstrel I just left "can not" as it is in master for now.

Reply:
You quote "will" here but it seems to still be "should" in the file?

Reply:
Pushed the quote.